tests/test_twopoint_difference.py

```diff
@@ -1019,10 +1018,13 @@ def test_median_func():
     arr = np.zeros(4 * 2 * 2).reshape(4, 2, 2)
     arr[:, 0, 0] = np.array([-1., -2., np.nan, np.nan])
     assert calc_med_first_diffs(arr)[0, 0] == -1
-@pytest.mark.skip("Used for local testing")
-def test_sigma_clip():
+
+
+# TODO: The input data for this is missing, need to find it
+@pytest.mark.xfail(reason="Missing Input Data")
+def test_sigma_clip(tmp_path, data_path):
```

Comment:
This test (and the other tests in this file requiring input data) where added by #174. If @mwregan2 could provide the input data used here then I can fully enable these. Otherwise I suggest removing this and related tests.